endpoints/citrus-camel/src/test/java/com/consol/citrus/camel/actions/CamelActionBuilderTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.consol.citrus.camel.actions;
+
+import java.util.Map;
+
+import com.consol.citrus.TestActionBuilder;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+/**
+ * @author Christoph Deppisch
+ */
+public class CamelActionBuilderTest {
+
+    @Test
+    public void shouldLookupTestActionBuilder() {
```

Comment:
test classes and methods should not be public, but that could be part of a cleanup. because if we start it here, we're kinda inconsistent with the rest of the code base.

Reply:
yes, let's do this all at once

Reply:
With this I would like to bring up the discussion to switch to JUnit Jupiter instead of TestNG for all unit tests. WDYT?

Reply:
I am personally exceptionally using JUnit Jupiter.. so, it's a "yes" for me ;)

Reply:
@christophd could you create a "citrus release 4.0" project and maybe allow me to add some issues there? not sure what the roles are called, but maybe you can make me an "admin"? I would then like to track the issues which this breaking change could target.